superset-frontend/src/views/CRUD/data/dataset/AddDataset/DatasetPanel/index.tsx

```diff
@@ -17,16 +17,45 @@
  * under the License.
  */
 import React from 'react';
-import { t, styled } from '@superset-ui/core';
+import { supersetTheme, t, styled } from '@superset-ui/core';
+import Icons from 'src/components/Icons';
 import { EmptyStateBig } from 'src/components/EmptyState';
 
+type DatasetPanelProps = {
+  tableName?: string | null | undefined;
```

Comment:
Do we need the `null | undefined` taking into account that the field is optional? Cannot just be :
`tableName?: string` ?

Reply:
Oh yeah good call! Corrected in [`this commit`](https://github.com/apache/superset/pull/21893/commits/d9501c64c1957fe94ca50d086ff676bbf7c8a6c5).

Reply:
Actually, this prop is `null` when a table isn't selected so I added it to the type in this [`commit`](https://github.com/apache/superset/pull/21893/commits/eb9a0d84c625fbbe98b7e9581aef9606ca6aea9e). Undefined wasn't needed here though, so still a good call, thanks! 😁

Reply:
heheh ok ok 😄